shared_workflow/shared.py

```diff
@@ -612,34 +614,53 @@ def submit_sl_script(
                 )
                 sys.exit()
 
-            update_db_cmd(process, status, mgmt_db_loc, srf_name, jobid, timestamp)
+            add_to_queue(
+                queue_folder,
+                run_name,
+                proc_type,
+                const.Status.queued.value,
+                job_id=jobid,
+            )
             return jobid
         else:
             print("An error occurred during job submission: {}".format(res[1]))
     else:
         print("User chose to submit the job manually")
 
 
-def update_db_cmd(process, status, mgmt_db_loc, srf_name, jobid, timestamp):
-    """Adds the command to update the mgmt db to the queue"""
-    # Update the db
-    if mgmt_db_loc is not None and os.path.isdir(mgmt_db_loc):
-        db_queue_path = os.path.join(mgmt_db_loc, "mgmt_db_queue")
-        cmd_name = os.path.join(db_queue_path, "%s_%s_q" % (timestamp, jobid))
-        cmd = (
-            "python $gmsim/workflow/scripts/management/"
-            "update_mgmt_db.py {} {} {} --run_name {}  --job {}".format(
-                mgmt_db_loc, process, status, srf_name, jobid
+def add_to_queue(
```

Comment:
any reason to put this function into the shared.py? 
because by doing so we create extra dependencies on a shared lib.